src/core/contexts.ts

```diff
@@ -15,12 +15,14 @@ type UpdateAtom = <Value, Update>(
 
 type StoreForProduction = [
   stateMutableSource: MutableSource<State>,
-  updateAtom: UpdateAtom
+  updateAtom: UpdateAtom,
+  commit: () => void
```

Comment:
Maybe type for Commit callback

Reply:
Do you have a better name? `Commit` can be too simple.

Reply:
`CommitCallback`